checker/tests/resourceleak/SocketIntoList.java

```diff
@@ -13,26 +14,34 @@ public void test1(List<@MustCall({}) Socket> l) {
     l.add(s);
   }
 
-  // :: error: type.argument
-  public void test2(List<Socket> l) {
+  public List<Socket> test2(@OwningCollection List<Socket> l) {
     // s is unconnected, so no error is expected when it's stored into the list.
     // But, if the list is unannotated, we do get an error at its declaration site
     // (as expected, due to #5912).
```

Comment:
Is this comment still accurate with the collections extension? (I don't think that it is.)